lib/publiccloud/ec2.pm

```diff
@@ -114,14 +114,38 @@ sub upload_img {
 
     die("Create key-pair failed") unless ($self->create_keypair($self->prefix . time, 'QA_SSH_KEY.pem'));
 
+    # AMI of image to use for helper VM to create/build the image on CSP.
+    # AMI is region specific also we need to use different AMI's for on-demand/BYOS uploads
+    # In case needed combination not covered by this hash c<PUBLIC_CLOUD_EC2_UPLOAD_AMI> variable
+    # will be used. Upload will fail without this variable
+    my $ami_id_hash = {
+      'us-west-1-byos' => 'ami-02b1a98531379c966',
```

Comment:
I always found it helpful to add a comment with the name, corresponding to that ami.
Like we had in the ec2utils.conf file https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/e05750beca067ab05b53ccacfea012923b931313/data/publiccloud/ec2utils.conf#L84

Reply:
make sense will do that